app/views/report/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1>OpenFlight Carbon Leaderboard</h1>
 <h3 class="light-text">Servers with the lowest CO<sub>2</sub> emissions</h3>
-<h4 class="light-text"><%= "Grouped by identical specs" if @grouped %><h4>
+<h4 class="light-text"><%= "Grouped by identical specs" if @grouped %></h4>
```

Comment:
The fact that this WASNT simply flagged as an error when I wrote it is baffling to me but I suppose there's probably someone out there who wants their entire webpage to be a header 4 within a header 4.